modules.py

```diff
@@ -0,0 +1,71 @@
+import time
+class Product:
+    __id = ''
+    __name = ''
+    __price = 0.0
```

Comment:
Delete lines 3-5 because they're unnecessary. `__id, __name, and __price` are always overwritten inside `__init__`. Ref: https://stackoverflow.com/questions/11007627/python-variable-declaration

Reply:
Also, is there a reason why you used two underscores before each name? That causes the interpreter to change e.g. `__id` to `_Product__id`. Ref: https://shahriar.svbtle.com/underscores-in-python